packages/tasks/src/model-libraries.ts

```diff
@@ -408,6 +408,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		docsUrl: "https://hezarai.github.io/hezar",
 		countDownloads: `path:"model_config.yaml" OR path:"embedding/embedding_config.yaml"`,
 	},
+	hfendpoints: {
```

Comment:
all the images should show up here: https://huggingface.co/models?other=hfendpoints (whisper + sglang one too)